src/Model/Contact.php

```diff
@@ -96,11 +96,11 @@ public static function getOStatusCountByGroupId($gid)
 	public static function createSelfFromUserId($uid)
 	{
 		// Only create the entry if it doesn't exist yet
-		if (dba::exists('contact', ['uid' => intval($uid), 'self'])) {
+		if (dba::exists('contact', ['uid' => $uid, 'self' => true])) {
```

Comment:
Oh, yeah. Indeed, that code was not working. Do you think that we should change dba.php to make this work? Should be an easy task.

Reply:
No, it's more explicit this way.